test/src/main/scala/scala/test/junit/JunitMultipleSuffixes.scala

```diff
@@ -0,0 +1,25 @@
+package scala.test.junit
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.junit.Assert._
+
+class JunitSuffixE2E {
+
+  @Test
+  def someTest: Unit = {
+  	println("Running E2E")
+  	assertEquals(1, 1)
+  }
+
+}
+class JunitSuffixIT {
+
+  @Test
+  def someTest: Unit = {
+  	println("Running IT")
+  	assertEquals(1, 1)
+  }
+
```

Comment:
can we add a class that does not match the pattern with a failing test and see that it isn't called?

I think the pattern thing is super dangerous, by the way, since it is very easy to not notice that your tests are not being called because they don't match the pattern. I'm not really sure we should have this feature and instead discover all the tests. We have, unfortunately, had test refactors that disabled huge groups of tests and we didn't notice which ones had been disabled for a while.